src/dev/run_integrate_locale_files.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { run } from './run';
+import { integrateLocaleFiles } from './i18n/integrate_locale_files';
+
+run(async ({ flags: { path } }) => {
```

Comment:
__question:__ can we make this look similar to what we have for `i18n_check` (with Listr and friends)?

Reply:
@azasypkin the order in which this tool works is not similar to `i18n_check` and actually >90% of all execution time is spent for default messages extraction, so I think we can't